pact/provider/core/src/main/java/org/arquillian/algeron/pact/provider/core/PactsReader.java

```diff
@@ -26,17 +27,25 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Map;
+import java.util.ServiceLoader;
 import java.util.stream.Collectors;
 
 /**
  * Reads provided Pacts from defined @ContractsSource or any annotation meta-annotated with @ContractsSource
+ * or configured in arquillian configuration file.
  */
 public class PactsReader {
 
+    public static final String PROVIDER = "provider";
+
     @Inject
     @SuiteScoped
     InstanceProducer<Pacts> pactsInstanceProducer;
 
+    @Inject
+    Instance<AlgeronProviderConfiguration> algeronProviderConfigurationInstance;
+
     public void readPacts(@Observes BeforeClass test) {
```

Comment:
Well if class is called `PactsReader` it seems natural that the method is called `readPacts`